modular_citadel/code/modules/reagents/reagent container/hypospraymkii.dm

```diff
@@ -135,7 +163,7 @@
 	var/mob/living/L
 	if(isliving(target))
 		L = target
-		if(!L.can_inject(user, 1))
+		if(!penetrates && !L.can_inject(user, 1)) //This check appears another four times, since otherwise the penetrating sprays will break in do_mob.
```

Comment:
This is why having it as a flag is better, because the flag already makes this check itself.

Reply:
I should make this check a function so that it fits nicely into the do_mob but I'm not familiar enough with it to be comfortable trying.